opentracing/ext/tags.go

```diff
@@ -0,0 +1,49 @@
+package ext
+
+import (
+	"github.com/opentracing/api-golang/opentracing"
+)
+
+var (
+	// RPCTargetService records target service name for an RPC call
+	RPCTargetService = &stringTag{"rpc.target"}
```

Comment:
Oh, another idea: we could record the `"rpc.peer.process_guid"` for implementations that expose one, then pass that guid over the wire along with the in-band context data. The advantage here would be a pointer – within the tracing system – to more comprehensive information about the peer than just the host:service:port. (I realize this probably sounds like a lot of work in a zipkin world, but wanted to mention it... Thankfully, given the nature of `ext/`, this can be either/or.